doc/release-notes/5.0-release-notes.md

```diff
@@ -0,0 +1,321 @@
+
+### A Multi-File, Zipped Download Optimization
+
+In this release we are offering an experimental optimization for the multi-file, download-as-zip functionality. If this option is enabled, instead of enforcing size limits, we attempt to serve all the files that the user requested (that they are authorized to download), but the request is redirected to a standalone zipper service running as a cgi executable. Thus moving these potentially long-running jobs completely outside the Application Server (Payara); and preventing service threads from becoming locked serving them. Since zipping is also a CPU-intensive task, it is possible to have this service running on a different host system, thus freeing the cycles on the main Application Server. The system running the service needs to have access to the database as well as to the storage filesystem, and/or S3 bucket.
+
+Please consult the scripts/zipdownload/README.md in the Dataverse 5 source tree.
+
+The components of the standalone "zipper tool" can also be downloaded
+here:
+
+https://github.com/IQSS/dataverse/releases/download/v5.0/zipper.zip
+
+### Updated File Handling
+
+Files without extensions can now be uploaded through the UI. This release also changes the way Dataverse handles duplicate (filename or checksum) files in a dataset. Specifically:
+
+- Files with the same checksum can be included in a dataset, even if the files are in the same directory.
+- Files with the same filename can be included in a dataset as long as the files are in different directories.
+- If a user uploads a file to a directory where a file already exists with that directory/filename combination, Dataverse will adjust the file path and names by adding "-1" or "-2" as applicable. This change will be visible in the list of files being uploaded.
+- If the directory or name of an existing or newly uploaded file is edited in such a way that would create a directory/filename combination that already exists, Dataverse will display an error.
+- If a user attempts to replace a file with another file that has the same checksum, an error message will be displayed and the file will not be able to be replaced.
+- If a user attempts to replace a file with a file that has the same checksum as a different file in the dataset, a warning will be displayed.
+- Files without extensions can now be uploaded through the UI.
+
+### Pre-Publish DOI Reservation with DataCite
+
+Dataverse installations using DataCite will have DOIs reserved as "draft" with DataCite when the Dataset is created, instead of the DOI reservation taking place locally at create time and the reservation and switch to "findable" taking place with DataCite at publish time. This allows the DOI to reserved earlier in the publication process.
```

Comment:
crap, I've typed, and lost a comment here twice in a row... there may be something wrong with this paragraph: it appears to be saying "dataverse will be reserving the DOI at create time, instead of reserving it at create time"? - maybe not (it says "locally" in the second part?) - but there may be simply TMI in it. I would leave the technical details (about "findable" and "draft" etc.) in the upgrade instruction further down; but only say 
"Dataverse installations using DataCite will be able to reserve the persistent identifiers for datasets ahead of publishing time." - or something along these lines?

Reply:
Updated text in https://github.com/IQSS/dataverse/commit/2ec68b34c4671203fc1360ff88d0de95989d61d6

I agree that at this point in the doc (release highlights) we can be less technical.